monkey-runner/plugin/src/main/groovy/com/novoda/monkey/TargetedMonkey.groovy

```diff
@@ -0,0 +1,24 @@
+package com.novoda.monkey
+
+import com.novoda.gradle.command.Monkey
+
+class TargetedMonkey extends Monkey {
+
+    def packageName
+    def logFileName
+
+    @Override
+    def getPackageName() {
```

Comment:
given `packageName` is not private you get public [getter and setter](http://groovy-lang.org/style-guide.html#_getters_and_setters) for free, therefore you don't need to re-define the getter here